src/test/resources/pages/instructorStudentListPageWithPicture.html

```diff
@@ -154,7 +154,7 @@
sPage?courseid=course3&amp;studentemail=CCSDetailsUiT.charlie.tmms@gmail.tmt&amp;user=instructorWith2Courses&amp;addComment=team">Comment on Team 2</a></li></ul></div></td></tr><tr id="student-c1.3"><td id="studentphoto-c1.3"><div class="profile-pic-icon-click align-center" data-link="/page/studentProfilePic?studentemail={*}&amp;courseid={*}&amp;user=instructorWith2Courses"><a class="student-profile-pic-view-link btn-link">View Photo</a><img src="" alt="No Image Given" class="hidden"></div></td><td style="display: table-cell;" id="studentsection-c1.0" class="hidden">None</td><td id="studentteam-c1.0.1">Team 2</td><td id="studentname-c1.3">Denny Charlés</td><td id="studentemail-c1.3">denny.c.tmms@gmail.tmt</td><td class="no-print align-center"><a data-original-title="View the details of the student" class="btn btn-default btn-xs student-view-for-test" href="/page/instructorCourseStudentDetailsPage?courseid=course3&amp;studentemail=denny.c.tmms@gmail.tmt&amp;user=instructorWith2Courses" title="" target="_blank" data-toggle="tooltip" data-placement="top"> View</a>&nbsp;<a data-original-title="Use this to edit the details of this student. <br>To edit multiple students in one go, you can use the enroll page: <br>Simply enroll students using the updated data and existing data will be updated accordingly" class="btn btn-default btn-xs student-edit-for-test" href="/page/instructorCourseStudentDetailsEdit?courseid=course3&amp;studentemail=denny.c.tmms@gmail.tmt&amp;user=instructorWith2Courses" title="" target="_blank" data-toggle="tooltip" data-placement="top"> Edit</a>&nbsp;<a class="btn btn-default btn-xs student-delete-for-test" href="/page/instructorCourseStudentDelete?courseid=course3&amp;studentemail=denny.c.tmms@gmail.tmt&amp;user=instructorWith2Courses" onclick="return toggleDeleteStudentConfirmation('course3','Denny Charlés')"> Delete</a>&nbsp;<a data-original-title="View all student's evaluations and feedbacks" class="btn btn-default btn-xs student-records-for-test" href="/page/instructorStudentRecordsPage?courseid=course3&amp;studentemail=denny.c.tmms@gmail.tmt&amp;user=instructorWith2Courses" title="" target="_blank" data-toggle="tooltip" data-placement="top"> All Records</a>&nbsp;<div class="dropdown inline"><a class="btn btn-default btn-xs dropdown-toggle" href="javascript:;" data-toggle="dropdown"> Add Comment</a><ul class="dropdown-menu align-left" role="menu" aria-labelledby="dLabel"><li role="presentation"><a target="_blank" role="menuitem" tabindex="-1" href="/page/instructorCourseStudentDetailsPage?courseid=course3&amp;studentemail=denny.c.tmms@gmail.tmt&amp;user=instructorWith2Courses&amp;addComment=student">Comment on Denny Charlés</a></li><li role="presentation"><a target="_blank" role="menuitem" tabindex="-1" href="/page/instructorCourseStudentDetailsPage?courseid=course3&amp;studentemail=denny.c.tmms@gmail.tmt&amp;user=instructorWith2Courses&amp;addComment=team">Comment on Team 2</a></li></ul></div></td></tr></tbody></table></div>
```

Comment:
This test is failing. There is a pixel value that need to be change to wildcard "{*}px", as different browser size will cause the pixel value to change. search for "px" to find it.